packages/trader/src/AppV2/Components/DraggableList/draggable-list-item.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { StandaloneGripDotsVerticalBoldIcon, StandaloneCircleMinusFillIcon } from '@deriv/quill-icons';
+
+type DraggableListItemProps = {
+    title: string;
+    leftIcon?: React.ReactNode;
+    rightIcon?: React.ReactNode;
+    onLeftIconClick?: () => void;
+    onRightIconClick?: () => void;
+}
+
+const DraggableListItem: React.FC<DraggableListItemProps> = ({
+    title,
+    leftIcon,
+    rightIcon,
+    onLeftIconClick,
+    onRightIconClick,
+}) => {
+    const default_left_icon = <StandaloneGripDotsVerticalBoldIcon iconSize='sm' />;
```

Comment:
i'm a bit confused :) props are in camel and variables inside are in snake. let's choose one :)

Reply:
This is because I decided to follow quill components approach, we are using camelCase for properties and I think it will cause confusion if you need to pass snake case instead. Since the point is to move these components to quill some day, I think it makes sense. The difference between variables and props is that variables are set and never practically touched again, while if you want to reuse the component in the future, you need to pass props most of the time.